install.sh

```diff
@@ -26,9 +27,9 @@ OPTIONS:
                             If this value is not set and the Kubernetes is not installed, this script will exit with 1.
     -n, --name              Name of Kubernetes cluster, default value: kind
         --kind-version      Version of the Kind tool, default value: v0.7.0
-        --node-num          The count of the cluster nodes,default value: 6
+        --node-num          The count of the cluster nodes,default value: 3
         --k8s-version       Version of the Kubernetes cluster,default value: v1.12.8
-        --volume-num        The volumes number of each kubernetes node,default value: 9
+        --volume-num        The volumes number of each kubernetes node,default value: 3
```

Comment:
seems 3 isn't enough?

Reply:
each node has 3 volumes

Reply:
Yeah, but often deploy with the system to test